pdr_backend/cli/test/test_cli_arguments_lake.py

```diff
@@ -0,0 +1,67 @@
+import os
+from unittest.mock import patch
+
+import pytest
+from enforce_typing import enforce_types
+
+from pdr_backend.cli.cli_arguments_lake import str_as_abspath
+from pdr_backend.cli.cli_module_lake import do_lake_subcommand
+
+
+@enforce_types
+def test_str_as_abspath():
+    abs_path = os.path.abspath("lake_data")
+    assert str_as_abspath("lake_data") == abs_path
+    assert str_as_abspath(os.path.abspath("lake_data")) == abs_path
+
+
+@enforce_types
+def test_timestr_args(capsys):
+    args = ["raw", "drop", "lake_data", "invalid_date"]
+
+    with pytest.raises(SystemExit):
+        do_lake_subcommand(args)
+
+    captured = capsys.readouterr()
+    assert "error: argument ST: invalid timestr value: 'invalid_date'" in captured.err
+
+    # drop does not recognize the end date as a valid argument
+    args = ["raw", "drop", "lake_data", "2021-01-01", "2021-01-02"]
+    with pytest.raises(SystemExit):
+        do_lake_subcommand(args)
+
+    captured = capsys.readouterr()
+    assert "error: unrecognized arguments: 2021-01-02" in captured.err
```

Comment:
thank you! great test/stub and assert

taggin @kdetry @KatunaNorbert for reference